jetstream/core/orchestrator.py

```diff
@@ -442,6 +449,8 @@ def _prefill_thread(self, idx: int):
       my_transfer_backlog = self._transfer_backlogs[idx]
       # The prefill thread can just sleep until it has work to do.
       request = self._prefill_backlog.get(block=True)
+      self._prefill_backlog_size_metric.set(self._prefill_backlog.qsize())
```

Comment:
qq, should we record the metrics in to places?

Reply:
I would assume this is the real prefill queue size during runtime @FanhaiLu1 right?

Reply:
L430 logs the prefill backlog queue size after a request added to the queue; L452 logs the prefill backlog queue size after a request remove/get from the queue.

Reply:
then I would assume we only need to add it in one place right?

Reply:
IIUC, @Bslabe123 is trying to collect the prefill queue size metric as the first step?